### PR TITLE
Misc. Documentation work and fix for demo

### PR DESCRIPTION
**Pull request by kvptkr**
These are the changes I made when I was trying to get the demo working right before + some misc. documentation changes

### CODE REVIEW

File: server/profileFormSubmission.js [hunk 1]
```diff
@@ -46,12 +46,13 @@ module.exports.saveProfile = async (event, callback )=> {
     s3Urls[0] = govId == true ? s3.getSignedUrl('putObject', s3Params(LICENSE_BUCKET, "licenses", profileId)) : '';
     s3Urls[1] = toolPic == true ? s3.getSignedUrl('putObject', s3Params(TOOLS_BUCKET, "tool pics", profileId)): '';
 
+    console.log(profileId);
     return {
       statusCode: 200,
       headers:{
         "Access-Control-Allow-Headers" : "Content-Type",
         "Access-Control-Allow-Methods" : "POST",
-        "Access-Control-Allow-Origin" : "http://localhost:3000"
```
Comment (nicolasabousawan) [inline server/profileFormSubmission.js:54]: I thought we are going to disable cors?

Comment (nicolasabousawan) [reply server/profileFormSubmission.js:54]: So this should just be "*"

**Review by nicolasabousawan — CHANGES_REQUESTED** [review]

File: server/profileFormSubmission.js (hunk 1 above)
Comment (kvptkr) [reply server/profileFormSubmission.js:54]: good catch

File: server/README.md [hunk 2]
```diff
@@ -88,19 +88,22 @@ pre-requiste: node.js is already installed on your machine
 ```
 
 ## Deploying the Application
-* If you wish to deploy, push to the branch 'deploy-to-heroku' and our pipeline will run
+If you wish to deploy, there are two potential avenues:
+1. Push to the branch 'deploy-to-heroku' and our pipeline will run
+2. Using the heroku UI, deploy the branch you want to test out
 * A demo can be accessed here: https://serene-ocean-22768.herokuapp.com/
+* Please announce that you're deploying to make sure you don't overwrite someone else's changes
```
Comment (christinajiyunlee) [inline server/README.md:95]: should we note somewhere to pull all main branch updates before deploying to mitigate this overwriting issue?

**Review by nicolasabousawan — APPROVED** [review]
As long as you adjust the "Access-Control-Allow-Origin" header, we should be good. Approving with the expectation that will be done

File: ui/src/Components/CreateProfile.jsx [hunk 3]
```diff
@@ -3,8 +3,8 @@ import { Button, Col, Form, FormGroup, Label, Input} from 'reactstrap';
 import NavBar from './NavBar';
 
 //Uncomment/Comment based on env
-const URL = "https://bixe448nsa.execute-api.us-west-1.amazonaws.com/dev/insertFormData";
-// const URL = "http://localhost:3000/dev/insertFormData";
+// const URL = "https://bixe448nsa.execute-api.us-west-1.amazonaws.com/dev/insertFormData";
+const URL = "http://localhost:3000/dev/insertFormData";
```
Comment (christinajiyunlee) [inline ui/src/Components/CreateProfile.jsx]: should we be updating this to the production URL?

File: server/README.md (hunk 2 above)
Comment (kvptkr) [reply server/README.md:95]: Hmm, I think maybe I wasn't clear here. The purpose of announcing is just to make sure that someone else isn't testing their changes out at the same time on Heroku and potentially thinking that their changes aren't working.

File: ui/src/Components/CreateProfile.jsx (hunk 3 above)
Comment (kvptkr) [reply ui/src/Components/CreateProfile.jsx]: 🥇  good catch

**Pull request merged**